ui/src/main/resources/edu/wpi/grip/ui/MainWindow.fxml

```diff
@@ -94,6 +94,23 @@
                     </accelerator>
                 </MenuItem>
             </Menu>
+            <Menu text="Tools">
+                <MenuItem text="Settings" onAction="#showProjectSettingsEditor">
+                    <graphic>
+                        <ImageView styleClass="menu-graphic">
+                            <fitWidth>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitWidth>
+                            <fitHeight>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitHeight>
+                            <image>
+                                <Image url="@icons/settings.png"/>
```

Comment:
Can you screenshot me what the deploy glyph looks like on your screen?
I don't understand.